### PR TITLE
update version

### PR DESCRIPTION
**Pull request by BelehovEgor**

### CODE REVIEW

**Review by djino97 — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ on:
   push:
     branches: [ develop ]
   pull_request:
-    branches: [ develop ]
+    branches: [ develop, master ]
```
Comment (TIove) [inline .github/workflows/main.yml:9]: For what?
In other services it is just develop

File: src/UserService.Data/UserRepository.cs [hunk 2]
```diff
@@ -204,11 +204,12 @@ public bool IsExistUser(Guid userId)
...
         }
 
-        public IEnumerable<DbUser> Search(string text)
+        public List<DbUser> Search(string text)
         {
-            return _provider.Users.Where(u => u.LastName.Contains(text, StringComparison.OrdinalIgnoreCase)
-                                            || u.FirstName.Contains(text, StringComparison.OrdinalIgnoreCase)
-                                            || u.MiddleName.Contains(text, StringComparison.OrdinalIgnoreCase));
+            return _provider.Users
+                                .ToList()
```
Comment (TIove) [inline src/UserService.Data/UserRepository.cs:210]: It looks strange cast to list two times? Is it really doesn't work without it?

File: .github/workflows/main.yml (hunk 1 above)
Comment (BelehovEgor) [reply .github/workflows/main.yml:9]: I made it like ProjectService

File: src/UserService.Data/UserRepository.cs (hunk 2 above)
Comment (BelehovEgor) [reply src/UserService.Data/UserRepository.cs:210]: first to list get all users, it is not good, but the condition that I want cannot be fulfilled without it

Comment (BelehovEgor) [reply src/UserService.Data/UserRepository.cs:210]: how I understand it, this request can't convert to sql request

**Review by TIove — APPROVED** [review]

**Pull request merged**